File: Easy/09_Palindrome_Number.py
class Solution(object):
    def isPalindrome(self, x):
        if(x<0):
            return False
        num = 0
        y = x
        while(y != 0):
            num = (num * 10) + (y%10)
            y = y // 10
        return (x == num)
        
print (3 // 2)    




# def isPalindrome(self, x):
    #     array = []
    #     y = x
    #     if(x < 0):
    #         return False
    #     while y != 0:
    #         array.append(y%10)
    #         y = (y / 10)
    #     left = 0
    #     right = len(array) - 1

    #     while left <= right:
    #         if(array[left] == array[right]):
    #             left += 1
    #             right -= 1
    #         else:
    #             return False
    #     return True

File: EasyJS/09_Palindrome_Number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  // const s = x.toString();
  // const reverse_s = s.split('').reverse().join('');
  // return true ? reverse_s == s : false
  let num = x;
  let reverse = 0;
  while (num > 0) {
    let digit = num % 10;
    reverse = reverse * 10 + digit;
    num = Math.floor(num / 10);
  }
  return reverse == x;
};
